WcaOnRails/config/locales/en.yml

```diff
@@ -117,7 +117,7 @@ en:
         pending_avatar: "After uploading a new avatar, you will have to wait for the Board to approve it."
       competition:
         competition_id_to_clone: "Optional. Can save you some typing if you're running a similar competition to a past competition."
-        name: "The full name of the competition including the year at the end (e.g., European Rubik's Cube Championship 2006). Be sure to capitalize."
```

Comment:
The example name was longer than the suggested 32 characters to I changed it to a competition name that passes the warnings.